vite.config.js

```diff
@@ -33,9 +33,7 @@ const css = {
       // remove this when we switch sass to use modern js api
       api: 'legacy',
       additionalData: `
-        @use 'sass:math';
-        @use 'sass:color';
-        @import 'src/scss/provider-variables.scss';
+        @use 'src/scss/provider-variables.scss' as *;
```

Comment:
From what I'm reading in the sass docs, doing a `use '' as *` is fine, as long as it's a stylesheets written by use: https://sass-lang.com/documentation/at-rules/use/#choosing-a-namespace